docs/ldml/tr35-numbers.md

```diff
@@ -1331,30 +1337,290 @@ For the formatting of number ranges, see <a href="#Number_Range_Formatting">Numb
+| a rule set name                                     | Perform the mathematical operation on the number, and format the result using the named rule set.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| a [Decimal Format Pattern](#Number_Format_Patterns) | Perform the mathematical operation on the number, and format the result using a [Decimal Format Pattern](#Number_Format_Patterns) with the specified pattern. The pattern must begin with 0 or #.                                                                                                                                                                                                                                                                                                                                                                                  |
+| nothing                                             | Perform the mathematical operation on the number, and format the result using the rule set containing the current rule, except: <ul><li>You can't have an empty substitution descriptor with a == substitution.</li> <li>If you omit the substitution descriptor in a >> substitution in a fraction rule, format the result one digit at a time using the rule set containing the current rule.</li> <li>If you omit the substitution descriptor in a << substitution in a rule in a fraction rule set, format the result using the default rule set for this formatter.</li></ul> |
+
+Whitespace is ignored between a rule set name and a rule set body, between a rule
+descriptor and a rule body, or between rules. If a rule body begins with an apostrophe,
+the apostrophe is ignored, but all text after it becomes significant (this is how you can
+have a rule's rule text begin with whitespace). There is no escape function: the semicolon
+is not allowed in rule set names or in rule text, and the colon is not allowed in rule set
+names. The characters beginning a substitution token are always treated as the beginning
+of a substitution token.
+
+The following `<ruleset>` and `<rule>` tags will be removed in the next release. They contain redundant information contained in `<rbnfRules>` to provide time to transition to `<rbnfRules>`. 
```

Comment:
This is an important line to note. The plan is to remove the older syntax after this next release.

Reply:
@richgillam There is now a header above this paragraph. It's now a little more prominent.